packages/shared/lib/sdk/sync.ts

```diff
@@ -565,7 +574,9 @@ export class NangoAction {
      * http = green
      * silly = light green
      */
-    public async log(...args: any[]): Promise<void> {
+    public async log(s: any, a?: { level: LogLevel }): Promise<void>;
+    public async log(s: string, ...args: [any, { level: LogLevel }]): Promise<void>;
```

Comment:
can we call `s` message and `a` options or something similar to make it slightly easier to read. Also shouldn't `level` be `level?`?

Reply:
definitely, got carried away when I found the working type :D